src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/close.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: 'close()'
-excerpt: 'Browser module: Browser.close method'
+excerpt: 'Browser module: close method'
```

Comment:
Shouldn't this not be documented? Considering that now the browser close is handled automatically by us and not exposed to the user?

Reply:
See: cdac40bb81b1f3142eb1b21dfddb29ec9b2a4dc9